src/App.tsx

```diff
@@ -1,7 +1,41 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string | null,
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: null,
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.namePressedButton);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.namePressedButton);
+  }
+
+  namePressedButton = (event: KeyboardEvent) => {
+    const pressedKey = event.key;
+
+    this.setState({ pressedKey });
+  };
+
+  render() {
+    const { pressedKey } = this.state;
+
+    return (
+      <div className="App">
+        { pressedKey ? (
+          <p className="App_message">
+            {`The last pressed key is [${pressedKey}]`}
+          </p>
+        ) : (
+          <p>Nothing was pressed yet</p>
```

Comment:
please add the `App__message` className for default message. it will fix the test
```suggestion
          <p className="App__message">Nothing was pressed yet</p>
```